example-config.yml

```diff
@@ -273,6 +273,29 @@ itinerary:
   # One fare will always be shown by default
   #defaultFareKey: electronicRegular
 
```

Comment:
Please also note somewhere that you need to 8 returned itineraries from OTP.

Reply:
Comment added in 7b39a505d39ad5b7e7ecd1d39ccc1afd22c8acff!